src/compile/data/summary.ts

```diff
@@ -48,14 +47,14 @@ export namespace summary {
 
           // add min/max so we can use their union as unaggregated domain
           const scale = model.scale(channel);
-          if (scale && useUnaggregatedDomain(scale, model, channel)) {
+          if (scale && scale.domain === 'unaggregated') {
```

Comment:
Yep.  And that's why we have to initialize many scale properties in init phase rather than just doing everything in the parse phrase.  In this PR, I'm just making `domain` consistent with others.  :)